lib/cisco_node_utils/cmd_ref/vpc.yaml

```diff
@@ -194,16 +194,16 @@ system_mac:
   set_value: "<state> system-mac <mac_addr>"
   default_value: ''
 
-system_priority: 
+system_priority:
```

Comment:
This is being handled in https://github.com/cisco/cisco-network-node-utils/pull/280